selftests/unit/test_utils_params.py

```diff
@@ -86,6 +86,96 @@ def testGetItemMissing(self):
     def testGetItem(self):
         self.assertEqual(self.params['image_size'], "10G")
 
+    def testCfgEnabled(self):
+        self.params["something"] = "no"
+        self.params["foobar"] = "yes"
+
+        self.assertEqual(False, params.is_cfg_enabled('something'))
+        self.assertEqual(False, params.is_cfg_enabled('notgiven'))
+        self.assertEqual(True, params.is_cfg_enabled('foobar'))
+
+    def testMultiParamNone(self):
+        self.assertEqual(0, len(self.params.get_multi_param("something")))
+
+    def testMultiParamOnlyOne(self):
+        self.params['something'] = 'foobar'
+
+        found = self.params.get_multi_param("something")
+        self.assertEqual(1, len(found))
+        self.assertEqual('foobar', found[0])
+
+    def testMultiParamMany(self):
```

Comment:
I remember you mentioned that you want to remove this method as `object_params` should suffice. If you want to keep it, please consider the name `get_range` or `get_range_value` as it'd be shorter and IMO fits well (we get range of named items).

Also if you intend this for serialization purposes only, I'd perhaps suggest to implement `iter_range` instead of `get` as for serialization we might not need the list. (and getting list from generator is simple)

Reply:
No, as @luckyh  pointed out, it can be replaced very easily with already existing functionality from `objects`.  I will also resolve this since the change where you commented is outdated.